app/bundles/PointBundle/Entity/Point.php

```diff
@@ -379,4 +385,14 @@ public function getRepeatable()
     {
         return $this->repeatable;
     }
+
+    public function getGroup(): ?Group
+    {
+        return $this->group;
+    }
+
+    public function setGroup(?Group $group = null): void
```

Comment:
```suggestion
    public function setGroup(?Group $group): void
```
I can't imagine why would anyone call a setter with no value.